CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # eslint-config-digitalbazaar ChangeLog
 
+## 1.9.0 - 2019-02-18
```

Comment:
Omit date, and I believe this is going to be a 2.0.0 breaking release?

Reply:
changed to 2.0.0 no date